src/test/datascience/export/exportToPython.vscode.test.ts

```diff
@@ -22,7 +22,7 @@ suite('DataScience - Export Python', function () {
     teardown(closeActiveWindows);
     suiteTeardown(closeActiveWindows);
     test('Export To Python', async () => {
-        const fileSystem = api.serviceContainer.get<IFileSystem>(IFileSystem);
+        const fileSystem = api.serviceContainer.get<IFileSystemNode>(IFileSystemNode);
```

Comment:
Maybe do this as a separate PR, but it would be nice if this test was made to run in web too.

Reply:
Yep, will look into this in next PR>